src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -1032,9 +1030,8 @@ fn managed_cluster_replica_name(index: u32) -> String {
 
 /// The type of finalization needed after an
 /// operation such as alter_cluster_managed_to_managed.
-pub enum FinalizationNeeded {
+pub enum NeedsFinalization {
     /// Wait for the provided duration before finalizing
-    In(Duration),
-    /// Finalization has already occurred
-    False,
+    Yes,
+    No,
 }
```

Comment:
Totally subjective so feel free to ignore, but this probably could just turn into a `bool` now.

Reply:
Yeah, there's a weird inversion in the function call where the function returns a value indicating whether or not the caller should take additional action, ie, whether the caller should then call the finalize method. I'm in favor of keeping this an enum as the bool doesn't convey any of this.